packages/flutter_tools/lib/src/build_system/targets/assets.dart

```diff
@@ -128,7 +142,14 @@ Future<Depfile> copyAssets(
           bool doCopy = true;
           switch (entry.value.kind) {
             case AssetKind.regular:
-              break;
+              if (entry.value.transformers.isNotEmpty) {
+                doCopy = !await assetTransformer.transformAsset(
+                  asset: content.file as File,
+                  outputPath: file.path,
+                  workingDirectory: environment.projectDir.path,
+                  transformerEntries: entry.value.transformers,
+                );
```

Comment:
Note for self: we should instead fail the build if transformation fails

Reply:
Addressed in [instead of logging, ToolExit when a transformation fails](https://github.com/flutter/flutter/pull/143815/commits/ba7b1d973e0656f7fe69b03ecf8914693cbc589b)